Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -2297,8 +2306,21 @@ def sub_main():
     # client's default_target_mailbox is the authorization source for the instance
     params['default_target_mailbox'] = args.get('target_mailbox',
                                                 args.get('source_mailbox', params['default_target_mailbox']))
-    client = EWSClient(**params)
+    try:
+        client = EWSClient(**params)
```

Comment:
can't we add it to the try-except below?

Reply:
the log writing function in the second try-except interferes with the process of the exception, so you can't track the exception properly

Reply:
I made some changes and it's now inside the second try-except